Gemfile

```diff
@@ -64,7 +64,7 @@ group :development, :test do
   gem 'cucumber'
   gem 'aruba'
   gem 'rake_shared_context'
-  gem 'conjur-cli', github: 'conjurinc/cli-ruby', branch: 'possum'
+  gem 'conjur-cli', '~> 6.0'
```

Comment:
Can we use `~> 6` instead? That way we don't have to update it again when we hit `6.1.0`

Reply:
Fixed.